javascript/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "my-dapp",
+  "version": "0.1.0",
+  "description": "JS DApp",
+  "dependencies": {
+    "ethers": "^5.5.4"
+  },
+  "devDependencies": {
+    "webpack": "^5.44.0",
+    "webpack-cli": "^4.7.2"
+  },
+  "scripts": {
+    "build": "webpack --entry ./src/index.js --mode production",
+    "start": "ROLLUP_HTTP_SERVER_URL=\"http://127.0.0.1:5004\" node src/index.js"
```

Comment:
Do we need to expose the use of the `ROLLUP_HTTP_SERVER_URL` here?

I was thinking in defining a default value inside `index.js` in case it's not provided, but that one can override it needed.

Reply:
Hmm, I don't know. This `start` is currently used for host-mode execution